controller/src/lib.rs

```diff
@@ -168,19 +170,63 @@ async fn dispatch_schedule_request(
     };
 }
 
+async fn dispatch_lock_request(
+    state: StateHandle,
+    lock_request: MessageWithResponseHandle<FetchBackendForLock>,
+) {
+    let mut response: FetchBackendForLockResponse = FetchBackendForLockResponse::NoBackendForLock;
+    if let Ok(backend) = backend_of_lock(
+        &state,
+        &lock_request.value.cluster,
+        &lock_request.value.lock,
+    ) {
+        if let Ok(schedule_response) = schedule_response_for_existing_backend(
+            &state,
+            lock_request.value.cluster.clone(),
+            backend,
+        ) {
+            response = schedule_response.try_into().expect(
+                "already checked that backend exists, \\
+						 so schedule_response should be convertible to FetchBackendForLockResponse",
```

Comment:
we found another place that rustfmt misses tabs

although in this case it might be legit -- I think these tabs are actually part of the string, since they don't match the indentation of the previous line (don't quote me on that, I forget the exact rules)